components/brave_vpn/resources/panel/vpn_panel.tsx

```diff
@@ -42,7 +42,7 @@ function App () {
         dark={vpnDarkTheme}
         light={vpnLightTheme}
       >
-        <PanelWrapper>
+        <PanelWrapper ref={() => getPanelBrowserAPI().panelHandler.showUI()}>
```

Comment:
Is there a reason this is in a ref callback rather than a `useEffect`?

Reply:
I thought it only need to be called only once after mounted because it will be called later whenever visibility is changed. So ref callback is used.
but As you mentioned `useEffect()`, using `useEffect()` is same behavior with previous impls. changed to use `useEffect()`. Thanks!

Reply:
Sweet, lgtm!